net-analyzer/nuttcp/files/nuttcp.confd

```diff
@@ -0,0 +1,4 @@
+# Config file for /etc/init.d/nuttcp
+
+# extra options (run nuttcp -h for a list of supported options)
+NUTTCP_OPTS="-S"
```

Comment:
Would be nice to explain what the default `-S` does, so that people know whether they need to change anything.

Reply:
A package has a man file, "-S" is described in it. But anyway it costs nothing adding a line to the conf file. Ok.